boards/nxp/mimxrt1180_evk/doc/index.rst

```diff
@@ -0,0 +1,235 @@
+|                    |            | boot time.                          |
++--------------------+------------+-------------------------------------+
+
+Supported Features
+==================
+
+The mimxrt1180_evk board configuration supports the hardware features listed
+below.  For additional features not yet supported, please also refer to the
+:ref:`mimxrt1170_evk` , which is the superset board in NXP's i.MX RT11xx family.
+NXP prioritizes enabling the superset board with NXP's Full Platform Support for
+Zephyr.  Therefore, the mimxrt1170_evk board may have additional features
+already supported, which can also be re-used on this mimxrt1180_evk board:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | counter                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+:zephyr_file:`boards/nxp/mimxrt1180_evk/mimxrt1180_evk_mimxrt1189_cm33_defconfig`
+
+Other hardware features are not currently supported by the port.
+
+
+Connections and I/Os
+====================
+
+The MIMXRT1180 SoC has six pairs of pinmux/gpio controllers.
+
++---------------+-----------------+---------------------------+
+| Name          | Function        | Usage                     |
++===============+=================+===========================+
+| GPIO_AON_04   | GPIO            | SW8                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_27    | GPIO            | LED                       |
++---------------+-----------------+---------------------------+
+| GPIO_AON_08   | LPUART1_TX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AON_09   | LPUART1_RX      | UART Console              |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The MIMXRT1180 SoC is configured to use SysTick as the system clock source,
+running at 240MHz. When targeting the M7 core, SysTick will also be used,
+running at 792MHz
+
+Serial Port
+===========
+
+The MIMXRT1180 SoC has 12 UARTs. One is configured for the console and the
+remaining are not used.
+
+Programming and Debugging
```

Comment:
We should add some blurb here about how to to load an image onto the CM7 core versus the CM33 core

Reply:
I have added detailed operation in Using J-Link and Using linkserver section, because there are some difference on two ways.